### PR TITLE
Fix GetLocationCommand output type parameter set and style issues

### PR DESCRIPTION
**Pull request by Meir017**
## PR Summary

Fixes the GetLocationCommand output type parameter-set from the non-existing `locationSet` to use the constant field `LocationParameterSet`.
Fix style issues in Navigation.cs.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

perhaps all of the parameter-set-names constants should be in a single place since many of then are used in multiple commands
maybe move them into the base class?

### CODE REVIEW

File: src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs [hunk 1]
```diff
@@ -361,9 +361,9 @@ internal override CmdletProviderContext CmdletProviderContext
     /// </summary>
     /// <remarks>
     /// </remarks>
-    [Cmdlet(VerbsCommon.Get, "Location", DefaultParameterSetName = "Location", SupportsTransactions = true, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113321")]
-    [OutputType(typeof(PathInfo), ParameterSetName = new string[] { "locationSet" })]
-    [OutputType(typeof(PathInfoStack), ParameterSetName = new string[] { "Stack" })]
+    [Cmdlet(VerbsCommon.Get, "Location", DefaultParameterSetName = locationSet, SupportsTransactions = true, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113321")]
+    [OutputType(typeof(PathInfo), ParameterSetName = new string[] { locationSet })]
```
Comment (Meir017) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs]: the output was set to a non existing `ParameterSetName`

**Comment by Meir017** [comment]
CodeFactor fails because of `Constants must start with an upper-case letter` but all over the project contants start with a lower-case letter so I wasn't sure about this

File: src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs [hunk 2]
```diff
@@ -881,15 +881,25 @@ protected override void ProcessRecord()
...
-    [Cmdlet(VerbsCommon.Push, "Location", DefaultParameterSetName = "Path", SupportsTransactions = true, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113370")]
+    [Cmdlet(VerbsCommon.Push, "Location", DefaultParameterSetName = pathSet, SupportsTransactions = true, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113370")]
     public class PushLocationCommand : CoreCommandBase
     {
         #region Command parameters
 
+        /// <summary>
+        /// The string declaration for the Location parameter set in this command.
+        /// </summary>
+        private const string pathSet = "Path";
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs]: All const names should start with upper case.
```suggestion
        private const string PathParameterSet = "Path";
```

**Review by iSazonov — COMMENTED** [review]
Please use our template for PR description.

**Review by iSazonov — COMMENTED** [review]
@Meir017 Please add more information in PR description about that you want fix. It is not clear wgy you remove `GetResolvedPaths` and `RenameItem`.

**Comment by iSazonov** [comment]
@Meir017 Please look test failures - something was broken in first commits.

**Comment by Meir017** [comment]
@iSazonov btw to remove the `// get` comments we could just use a regex replace:
replace: `} // ([A-Za-z .]*)`
to: `}`
but I'm guessing it's out of scope for this issue

**Comment by Meir017** [comment]
@iSazonov Done

**Comment by iSazonov** [comment]
@Meir017 Thanks for your contribution!

Could you please look CodeFactor issues https://www.codefactor.io/repository/github/powershell/powershell/pull/8324 ?

**Comment by Meir017** [comment]
many of the changes are about the file `src\System.Management.Automation\engine\Attributes.cs` not sure about them

**Comment by vexx32** [comment]
Don't worry too much about anything that's in code files you haven't touched; just focus on the handful that are in or around the code you've modified. 😄

**Comment by iSazonov** [comment]
Restart CI-macos.

**Review by TravisEz13 — APPROVED** [review]

**Comment by iSazonov** [comment]
@Meir017 Thanks for your contribution!

**Pull request merged**